src/main/java/uk/gov/cshr/controller/account/email/ChangeEmailController.java

```diff
@@ -110,7 +111,7 @@ public String verifyEmail(@PathVariable String code, Authentication authenticati
         if (isWhitelisted(newDomain)) {
             log.debug("New email is whitelisted: oldEmail = {}, newEmail = {}", identity.getEmail(), emailUpdate.getEmail());
             try {
```

Comment:
As discussed, ensure we are consistent with the super seeding.  All scenarios should have if domain is tokenised then treat as tokenised user otherwise treat as whitelisted.

Reply:
We'll be handling whitelisted domains, before tokenised.